pallets/bridges/nomad/da-bridge/src/mock.rs

```diff
@@ -117,9 +102,10 @@ impl ExtBuilder {
 	}
 
 	pub(crate) fn build(self) -> sp_io::TestExternalities {
-		let mut t = frame_system::GenesisConfig::default()
-			.build_storage::<Test>()
-			.expect("Frame system builds valid default genesis config");
+		let mut t = RuntimeGenesisConfig::default()
+			.system
+			.build_storage()
+			.unwrap();
```

Comment:
Nothing important but would be nice if we used the old `expect` instead of unwrap,